app/workers/rapid_ready_for_decision/form526_base_job.rb

```diff
@@ -14,10 +16,15 @@ class Form526BaseJob
     # https://github.com/mperham/sidekiq/issues/2168#issuecomment-72079636
     sidekiq_options retry: 8
 
+    class NoRrdProcessorForClaim < StandardError; end
+
     def perform(form526_submission_id)
       form526_submission = Form526Submission.find(form526_submission_id)
 
       begin
+        RapidReadyForDecision::Constants.processor_class(form526_submission)
```

Comment:
Seems like since it's not switching yet this could go into a future PR, but also doesn't hurt anything.

Reply:
I added this line so that current tests will run the new code, otherwise there might be a low test code coverage error.